web/app/forms.py

```diff
@@ -79,3 +80,10 @@ class TournamentPlayerForm(Form):
     dob = StringField("Date of Birth", validators=[Required()])
 
     submit = SubmitField()
+
+class MatchResultsForm(Form):
+    player_1_name = StringField("Black's Name")
```

Comment:
AGA is trying to depreciate temporary ids. Players should sign up at the time of the tournament or ahead of time. This makes everything after the tournament go 100% faster.